### PR TITLE
Content validation script

### PR DESCRIPTION
**Pull request by Zebedeusz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,12 +122,32 @@ More information on the content validation for your custom data types can be fou
...
 
 where `schema_path` is a full path to the data type json file to validate.
 
+Furthermore, to validate data type values against generated `flat.json` file (or any file with the same structure), following script may be used:
+
+`npm run validate-content <content_validation_cases_path> <schema_path>`
+
+where:
+
+- `content_validation_cases_path` is a full path to json file with validation cases having following structure:
```
Comment (kebetsi) [inline README.md:137]: "... to a JSON file ... "

File: README.md [hunk 2]
```diff
@@ -122,12 +122,32 @@ More information on the content validation for your custom data types can be fou
...
+
+```json
+  [
+    {
+        "type": "absorbed-dose/gy",
+        "content": 45,
+        "expected": "success"
```
Comment (kebetsi) [inline README.md:144]: maybe provide an example for failure as well?

File: README.md [hunk 3]
```diff
@@ -122,12 +122,32 @@ More information on the content validation for your custom data types can be fou
...
+    }
+  ]
+```
+
+- `schema_path` is a full path to the data type json file to validate. Optional - if not provided, generated `flat.json` will be used.
```
Comment (kebetsi) [inline README.md:149]: good move on the default

File: src/validate.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+if(!process.argv[2]) {
+    console.error('Json file with validation cases not provided');
```
Comment (kebetsi) [inline src/validate.js:2]: JSON

File: README.md (hunk 1 above)
Comment (Zebedeusz) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (Zebedeusz) [reply README.md:144]: Done

File: src/validate.js (hunk 4 above)
Comment (Zebedeusz) [reply src/validate.js:2]: Done

**Pull request merged**